helmfile.d/snippets/defaults.yaml

```diff
@@ -694,12 +694,6 @@ environments:
         obj:
           provider:
             type: disabled
-          bucket:
```

Comment:
Not sure how values-changes behave with additions and relocations in combination (assuming that some values repos might have this and others might not). Therefore I would suggest to leave them in place and just rename to `buckets`. The nonexistence of these is the cause of my error as previously commented.